lib/flight_scheduler/configuration.rb

```diff
@@ -60,12 +62,37 @@ class Configuration
       {
         name: :node_type,
         env_var: true
+      },
+      {
+        name: :lower_ephemeral_port,
```

Comment:
Why is this named "ephemeral"? Wouldn't something like "stepd_port_range_start" be clearer?

Reply:
https://github.com/openflighthpc/flight-scheduler-daemon/pull/28/commits/c8e8bf4c58af16b8eb58d6c1ac0a5d4f55d18dd4